docs/development/release.md

```diff
@@ -90,8 +90,12 @@ When creating a backport, a minor version base branch should already exist on Gi
 ```bash
 # check out existing minor version base branch
 git checkout release-1.14.x
+# pull to ensure you have the latest changes
+git pull origin release-1.14.x
 # create new release branch for backport
 git checkout -b release-1.14.1
+# cherry pick the commits (or single squashed commit of changes) relevant to the backport
+git cherry-pick [replace-with-commit-SHA]
```

Comment:
For completeness, should we mention to `git push` when you're done cherry-picking?

```suggestion
git cherry-pick [replace-with-commit-SHA]
# push up the changes to the release branch
git push
```

Reply:
Yeah, that probably is good to add